src/extensions/unitstable/Units/UnitsTable.tsx

```diff
@@ -130,7 +130,11 @@ const UnitsTable: FunctionComponent<Props> = (props) => {
             return sortMetricValues(recordA, recordB, r.sortAscending)
         })
     }
-    const selectedUnitIdsLookup: {[key: string]: boolean} = selection.selectedUnitIds.reduce((m, uid) => {m[uid + ''] = true; return m}, {} as {[key: string]: boolean})
+    const selectedUnitIdsLookup: {[key: string]: boolean} = (selection.selectedUnitIds || []).reduce((m, uid) => {m[uid + ''] = true; return m}, {} as {[key: string]: boolean})
+    const mergeGroupForUnitId = (unitId: number) => {
+        const mergeGroups = (sorting.curation || {}).mergeGroups || []
+        return mergeGroups.filter(g => (g.includes(unitId)))[0] || null
```

Comment:
Assumes that a unit will appear in exactly one or zero merge groups. (Probably we will have maintained the list of merge groups so that it has this property--but that's emergent behavior from the way we're maintaining the list, rather than an enforced guarantee, so I wanted to note it.)